chart/assets/operations/job_moving/move_auctioneer.yaml

```diff
@@ -4,6 +4,9 @@
   path: /instance_groups/name=scheduler:after
   value:
     name: auctioneer
+    azs:
+    - z1
+    - z2
```

Comment:
can you explain what this will do by default please?

Reply:
If there is no the azs property, when enable the multi_az feature the auctioneer pod name is not with `-z*`,like auctioneer-0.  If the sizing instances is set to 1, other components will have 3 pods(1 pod in each zone) but auctioneer will have only 1 instance.